frontend/src/utils/fetch.tsx

```diff
@@ -92,6 +92,9 @@ export function queryParams(obj: Record<PropertyKey, unknown>): string {
           throw new Error(`Failed to encode key/value pair: ${key}=${val}`)
         }
         break
+      case "undefined":
+        /* Don't add any query param if value is undefined */
+        break
```

Comment:
I believe my suggestion above removes the need for this

Reply:
This is still needed because the min and max age use undefined to signal that a min and max age isn't specified. And in general, I decided that any variable that has a value of undefined should just be ignored rather than error. I suppose I could look at changing `...filters` to something that avoids including properties that are undefined but I don't think it's worth the effort.

Reply:
Maybe that Ramda library has something for this? I'm not familiar with it.

Reply:
Maybe, but for now this is fine